### PR TITLE
Yuhsuan/1397 fixed fitting

### PR DESCRIPTION
**Pull request by YuHsuan-Hwang**
**Description**
This PR is for the fifth item of #1397: fixing initial values
* add buttons for fixing initial values
* show `(fixed)` in the fitting results

corresponding backend PR: https://github.com/CARTAvis/carta-backend/pull/1183

**Checklist**
- [x] changelog updated / ~no changelog update needed~
- [x] ~~e2e test passing~~ / added corresponding fix
- [X] ~protobuf updated to the latest dev commit~ / no protobuf update needed

### CODE REVIEW

**Comment by YuHsuan-Hwang** [comment]
@kswang1029 fixed issues mentioned in the backend PR.

**Review by kswang1029 — APPROVED** [review]
this looks good. all e2e tests are passing. I will add related e2e tests later.

File: src/stores/ImageFittingStore.ts [hunk 1]
```diff
@@ -127,11 +130,15 @@ export class ImageFittingStore {
         log += this.getRegionInfoLog(regionId, fovInfo) + "\n";
         log += fittingLog + "\n";
 
-        const toFixFormat = (param: string, value: number | string, error: number, unit: string): string => {
-            return `${param} = ${typeof value === "string" ? value : value?.toFixed(6)} +/- ${error?.toFixed(6)}${unit ? ` (${unit})` : ""}\n`;
+        const toFixFormat = (param: string, value: number | string, error: number, unit: string, fixed: boolean): string => {
+            const valueString = typeof value === "string" ? value : value?.toFixed(6);
+            const errorString = fixed ? "" : " +/- " + error?.toFixed(6);
```
Comment (veggiesaurus) [inline src/stores/ImageFittingStore.ts]: can we use the unicode [plus-minus](https://www.compart.com/en/unicode/U+00B1#:~:text=%E2%80%9C%C2%B1%E2%80%9D%20U%2B00B1%20Plus%2DMinus%20Sign%20Unicode%20Character) sign instead? `±`

**Review by veggiesaurus — CHANGES_REQUESTED** [review]
just one minor change (using `±` instead of `+/-`, if @kswang1029 agrees?)

File: src/stores/ImageFittingStore.ts (hunk 1 above)
Comment (kswang1029) [reply src/stores/ImageFittingStore.ts]: I am ok with this. 👍

Comment (YuHsuan-Hwang) [reply src/stores/ImageFittingStore.ts]: changed to unicode plus-minus sign for both image fitting and z profile fitting

**Comment by kswang1029** [comment]
@YuHsuan-Hwang I think I need a fix in the e2e test due to the unicode +- change. So that's why I re-request myself a review.

**Review by kswang1029 — APPROVED** [review]
re-approving this PR with a fix in the e2e tests (as a branch to be merged when this PR is merged) due to the change of the unicode ±.

**Comment by kswang1029** [comment]
@YuHsuan-Hwang @veggiesaurus @confluence is it possible to get the image fitting PR pair merged soon?
